ydb/core/ymq/base/queue_attributes.cpp

```diff
@@ -76,6 +76,8 @@ TQueueAttributes TQueueAttributes::FromAttributesAndConfig(const THashMap<TStrin
             }
             result.Error = &NErrors::INVALID_PARAMETER_VALUE;
             break;
+        } else if (name == "FolderID") {
+            // NOTE: FolderID is used to override folder id from service account on queue creation
```

Comment:
Why are you not putting FolderID in result?

Reply:
Fixed